src/conn.rs

```diff
@@ -87,6 +89,14 @@ pub struct Conn {
     // the schema changes. #315.
 }
 
+// TODO: add lookup methods.
+// TODO: I'm not super happy with this abstraction; it's basically a limited `InProgress`. Can
```

Comment:
Yeah, this abstraction doesn't seem correct at all.

Reply:
The reasoning behind it is: what's the shared aspect of `Conn` and `InProgress`? Well, it's a thing against which I can run queries. This comment is actually an orphan — it really describes `QueryableTransaction`, which is the bit I'm not super happy with.